assets/src/components/propertiesPanel/crowdingDiagram.tsx

```diff
@@ -0,0 +1,110 @@
+import React from "react"
+import { crowdingIcon } from "../../helpers/icon"
+import { Vehicle } from "../../realtime"
+
+const extendedOccupancyStatus = (vehicle: Vehicle): string => {
```

Comment:
Is the list of potential occupancy statuses defined in a type anywhere? If it were, it'd make it easier to make sure that any case over all the options doesn't miss a status.